examples/Thermostat/Client/Program.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using Igloo;
+using Microsoft.Extensions.Logging;
+using System.CommandLine;
+
+using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+    builder
+        .AddSimpleConsole()
+        .AddFilter("IceRpc", LogLevel.Information));
+
+// Create a client connection to the cloud server.
+await using var connection = new ClientConnection(
+    new Uri("icerpc://localhost"),
+    logger: loggerFactory.CreateLogger<ClientConnection>());
+
+Pipeline pipeline = new Pipeline()
+    .UseDeadline(TimeSpan.FromSeconds(3))
+    .UseLogger(loggerFactory)
+    .Into(connection);
+
+var proxy = new ThermostatProxy(pipeline);
```

Comment:
I think it would be clearer to rename "proxy" to "thermostat". 

I think `thermostat.ChangeSetPointAsync()` vs `proxy.ChangeSetPointAsync()`.

Reply:
I agree I think it's cleaner. If we wanted to keep the idea of it being a proxy everywhere we could also do `thermostatProxy`

Reply:
I renamed it to thermostatProxy. `thermostat` would be more appropriate if we put it in an `IThermostat` variable.